src/modules/keyboardmanager/KeyboardManagerEditor/Resources.resx

```diff
@@ -180,13 +181,13 @@
     <value>Select the key you want to change (Key) and then the key or shortcut you want it to become (Mapped To).</value>
   </data>
   <data name="EditKeyboard_InfoExample" xml:space="preserve">
-    <value>For example, if you want to press A and get "Ctrl+C", key "A" would be your "Key" column and the shortcut "Ctrl+C" would be your "Mapped To" column.</value>
+    <value>Example of a remapping: Press A and get "Ctrl+C", "A" would be your "Physical Key" and "Ctrl+C" would be your "Mapped To" command.</value>
   </data>
   <data name="EditShortcuts_Info" xml:space="preserve">
     <value>Select the shortcut you want to change (Shortcut) and then the key or shortcut you want it to invoke (Mapped To).</value>
   </data>
   <data name="EditShortcuts_InfoExample" xml:space="preserve">
-    <value>For example, if you want to press "Ctrl+C" and get "Alt" only on Microsoft Edge, "Ctrl+C" would be your "Shortcut" column, the key "Alt" would be your "Mapped To" column, and "MSEdge" would be your "Target App" column. If no target app is entered, it will apply globally. The name must be the process name and not the app name.</value>
+    <value>Example of a remapping: Press "Alt+T" and get "Ctrl+T" only on Microsoft Edge, "Alt+T" would be your "Physical Shortcut", "Ctrl+T" would be your "Mapped To", and "MSEdge" (exe name) would be your "Target App". Note, if you remapped a physical key, that will influence the "Physical Shortcut".</value>
```

Comment:
@crutkas what do you want to say with "Note, if a physical key, that will influence the "Physical shortcut"?

For me this is a bit confusing.

Reply:
Remapped keys are maintained when used in (bigger) shortcuts